.travis.yml

```diff
@@ -1,11 +1,19 @@
+rvm:
+  - ruby-2.3.5-p0
+
 sudo: required
 services:
   - docker
 
+before_install:
+  - gem install bundler -v 1.15.4
+
 ### Skip the default install steps
```

Comment:
Since you've removed the `install: true` this comment is useless now.

Reply:
install is still true. It just has something to run in it.

Reply:
Well in the Travis doc https://docs.travis-ci.com/user/customizing-the-build#Skipping-the-Installation-Step it says that passing true means you're doing nothing so basically the comment was for that which is not the case anymore.

Reply:
Ah, you're talking about the comment. I was looking at the `before_install`.
Ok, I'll remove it.

Reply:
Done